### PR TITLE
Make checkPendingAndContinue protected to handle deserialisation crashes

### PR DESCRIPTION
**Pull request by arunreddy10**
#### Issue Reference
Refer the issue for this change

#### Change Summary
A brief description of the change.

#### Implementation Summary
A brief but thorough description of the changes put in place to address the issue.

#### How to Test
Detailed *list* of what to test and how to test it. Including all edge cases.

### CODE REVIEW

File: batching/build.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ android {
         minSdkVersion 14
         targetSdkVersion 28
         versionCode 1
-        versionName "1.3.9"
+        versionName "1.3.10"
```
Comment (anirudhramanan) [inline batching/build.gradle:41]: Can you also bump up the versions for other modules ? to keep versions in sync across

Comment (arunreddy10) [reply batching/build.gradle:41]: done

**Review by anirudhramanan — APPROVED** [review]
